common/src/main/java/com/wynntils/models/lootrun/LootrunModel.java

```diff
@@ -107,6 +111,11 @@ public class LootrunModel extends Model {
     //                        ÀÀÀ§7Challenges Completed: §f7
 
     private static final Pattern LOOTRUN_FAILED_PATTERN = Pattern.compile("[À\\s]*§c§lLootrun Failed!");
+    private static final Pattern CHALLENGE_FAILED_PATTERN = Pattern.compile("À.*?§c§lChallenge Failed!");
+
+    private static final Pattern MISSION_COMPLETED_PATTERN = Pattern.compile("[À\\s]*§b§lMission Completed");
+    private static final Pattern ADVANCED_MISSION_PATTERN = Pattern.compile("À.*?§b§l");
+    private static final Pattern MISSION_PATTERN = Pattern.compile(".*?§.");
```

Comment:
I would have really liked tests for this in TestRegex, for two reasons. This is really not a strict regex, I fear it will match too much. If I can see the missions texts, I can either help with a stricter regex, or I can see and accept it, as is.

Reply:
Okay, I see.. These are not real patterns, but a "builder" for the mission types. It might be better not to call them "patterns" then, and keep them as a string, or at the very least, comment that they are not used as declared here.